src/elements/content-uploader/ItemName.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import './ItemName.scss';
+
+interface ItemNameProps {
```

Comment:
Can we export this to retain the pattern with the rest of the components, knowing its not used outside of this component